pkg/controller/shared/authorization_informers.go

```diff
@@ -35,7 +35,7 @@ func (f *clusterPolicyInformer) Informer() cache.SharedIndexInformer {
 		return informer
 	}
 
-	lw := f.customListerWatchers.GetListerWatcher(kapi.Resource("clusterpolicies"))
+	lw := f.customListerWatchers.GetListerWatcher(authorizationapi.Resource("clusterpolicies"))
```

Comment:
To me `kapi.Resource("clusterpolicies")` == upstream, and if `nil` the code fallbacks to origin.

`authorizationapi.Resource("clusterpolicies")` == always use origin?

I am not familiar enough with this part of the code to say.

@deads2k ^ ?